### PR TITLE
SALTO-5952: Support GlobalStatuses for objectScehmas

### PR DESCRIPTION
**Pull request by idoamit8**
In this PR I support GlobalStatuses for objectSchemas. 

---

_Additional context for reviewer_
ObjectTypes can reference three kinds of statuses:
1. Status
2. ObjectSchema-specific statuses
3. Global objectSchema statuses

Until now, we supported types 1 and 2. With this PR, we will also support type 3.

---
_Release Notes_: 
_Jira Adapter:_
* Added support for global objectSchema statuses.

---
_User Notifications_: 
_Jira Adapter:_
* Added support for global objectSchema statuses.

### CODE REVIEW

File: packages/jira-adapter/src/config/api_config.ts [hunk 1]
```diff
@@ -2417,6 +2417,32 @@ const JSM_DUCKTYPE_TYPES: JiraDuckTypeConfig['types'] = {
...
   },
+  ObjectSchemaGlobalStatus: {
+    request: {
+      url: '/gateway/api/jsm/assets/workspace/{workspaceId}/v1/config/statustype',
+    },
+    transformation: {
+      dataField: '.',
+      fieldsToHide: [{ fieldName: 'id' }],
+      serviceIdField: 'id',
+      fieldsToOmit: [{ fieldName: 'objectSchemaId' }],
```
Comment (IdoZakk) [inline packages/jira-adapter/src/config/api_config.ts:2428]: What is the objectSchemaId?

**Review by IdoZakk — COMMENTED** [review]
🪅 
Might be worth to open a low priority ticket to change the category from number to text
Please add E2E and noise reduction

File: packages/jira-adapter/src/config/api_config.ts (hunk 1 above)
Comment (idoamit8) [reply packages/jira-adapter/src/config/api_config.ts:2428]: its just null because its global

**Review by IdoZakk — APPROVED** [review]
💯 
Please merge after the noise reduction is merged

**Pull request merged**